src/commands/dockerInstaller.ts

```diff
@@ -84,3 +79,30 @@ export class MacDockerInstaller extends DockerInstallerBase {
         terminal.show();
     }
 }
+
+export async function showDockerInstallNotification(): Promise<void> {
+    const installMessageLinux = localize('vscode-docker.commands.dockerInstaller.installDockerInfo', 'Docker is not installed. Would you like to learn more about installing docker?');
+    const installMessageNonLinux = localize('vscode-docker.commands.dockerInstaller.installDocker', 'Docker Desktop is not installed. Would you like to install?');
+    const learnMore = localize('vscode-docker.commands.dockerInstaller.learnMore', 'Learn more')
+    const install = localize('vscode-docker.commands.dockerInstaller.install', 'Install')
+    const osProvider: LocalOSProvider = new LocalOSProvider();
+    const isLinux = osProvider.os === 'Linux' && !osProvider.isMac;
+
+    const dockerInstallMessage = isLinux ? installMessageLinux : installMessageNonLinux;
+    const confirmationPrompt: vscode.MessageItem = isLinux ? { title: learnMore } : { title: install };
+    const response = await vscode.window.showInformationMessage(dockerInstallMessage, ...[confirmationPrompt]);
+    if (response) {
```

Comment:
Just to verify, if the user hits X it results in `response` being `undefined`, hence this block is skipped--correct?

Reply:
Yes that is right.